activesupport/lib/active_support/core_ext/date_and_time/calculations.rb

```diff
@@ -92,15 +92,28 @@ def years_since(years)
     end
 
     # Returns a new date/time at the start of the month.
+    #
+    #   today = Date.today # => Thu, 18 Jun 2015
+    #   today.beginning_of_month # => Mon, 01 Jun 2015
+    #
     # DateTime objects will have a time set to 0:00.
+    #
+    #   today = DateTime.today # => Thu, 18 Jun 2015 15:23:13 UTC +00:00
```

Comment:
thanks for the observation @repinel I hadn't noticed it, I updated the file with `Time.current` similar to the documentation of `DateAndTime::Calculations#next_week` (more below)